socorro/statsd_metrics.yaml

```diff
@@ -125,6 +125,30 @@ socorro.processor.es.save_processed_crash:
   description: |
     Timer for how long it takes to save the processed crash to Elasticsearch.
 
+socorro.processor.legacy_es.crash_document_size:
+  type: "histogram"
+  description: |
+    Size of crash document. In bytes.
+
+socorro.processor.legacy_es.index:
+  type: "histogram"
+  description: |
+    Total time it took to index the crash document in Elasticsearch.
+
+socorro.processor.legacy_es.indexerror:
+  type: "incr"
+  description: |
+    Counter for errors when indexing a document in Elasticsearch.
+
+    Tags:
+
+    * ``error``: the error code indicating what happened
+
+socorro.processor.legacy_es.save_processed_crash:
+  type: "timing"
+  description: |
+    Timer for how long it takes to save the processed crash to Elasticsearch.
```

Comment:
I'm pretty sure we only needed to add `socorro.processor.legacy_es.save_processed_crash` because that's the only one that's composed and emitted by the processor:

https://github.com/mozilla-services/socorro/blob/8d6449d2bb01fc7a0afa32a178e7773a6735d8aa/socorro/processor/processor_app.py#L215-L218

When you tested this, did you hit errors that caused you to add the other three metrics?

Reply:
I misunderstood the original request, i'll reduce it to the single new metric